src/playercards/CardsThatSealTokens.ttslua

```diff
@@ -0,0 +1,154 @@
+--[[ Library for cards that seal tokens
+This file is used to add sealing option to cards' context menu.
+Valid options (set before requiring this file):
+SHOW_READ_BAG       -- boolean
```

Comment:
Individual descriptions on these fields would be helpful.  It's also a bit unclear how they interact, so an example or two would be good (fine to reference the existent cards for it, just cite relevant examples for each field)